tests/PySys/monitoring/monitoring_with_simulated_messages/run.py

```diff
@@ -0,0 +1,116 @@
+from pysys.basetest import BaseTest
+
+import time
+import json
+
+"""
+Validate tedge-dm-agent  messages that are published
+on tedge/measurements
+
+Given a configured system
+When we start the tedge-dm-agent with sudo in the background
+When we start tedge sub with sudo in the background
+When we start two publishers to publish the simulated collectd messages
+Wait for couple of seconds to publish couple of batch of messages
+Then we kill tedge sub with sudo as it is running with a different user account
+Then we validate the  messages in the output of tedge sub,
+
+"""
+
+
+class MonitoringWithSimulatedMessages(BaseTest):
+    def execute(self):
+        tedge = "/usr/bin/tedge"
+        sudo = "/usr/bin/sudo"
+
+        # stop collecd to avoid mixup of messages
+        collectd = self.startProcess(
+            command=sudo,
+            arguments=["systemctl", "stop", "collectd"],
+            stdouterr="collectd",
+        )
+
+        collectd_mapper = self.startProcess(
+            command=sudo,
+            arguments=["systemctl", "start", "tedge-dm-agent"],
+            stdouterr="collectd_mapper",
+        )
```

Comment:
Same comment as in the other tests. Move the above calls to a `setup` function and stop dm-agent in cleanup.